error.go

```diff
@@ -232,6 +232,7 @@ type Error struct {
 
 	HTTPStatusCode    int               `json:"status,omitempty"`
 	Msg               string            `json:"message"`
```

Comment:
We might want to add `omitempty` to `Msg` too - right now the error will look like
```
{
  "status":400,
  "message":"",
  "developer_message":"Some fields in the request were invalid.",
  "request_id":"req_...",
  "type":""}
```

Reply:
If it doesn't affect deserialization let's leave it as is.